### ISSUE TITLE
Add Animation settings

### ISSUE THREAD

**Issue by Merlin-CK** [ubershmekel/redditp#86]

**Comment by ubershmekel** [comment]
I agree it's a bit slow and sloppy. I wonder if I should just drop the animation entirely.

**Comment by ubershmekel** [comment]
Just reduced the animation from 1000 to 100 ms. Maybe this is good enough? Would love to hear feedback on how it feels.

https://github.com/ubershmekel/redditp/commit/ceaf2dd05424bd739ea508bda91cf749755d908c
